src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -166,29 +170,20 @@ private void initializeCommandGroups() {
         new SequentialCommandGroup(
             new DeployIntake(intakeSubsystem, cdsSubsystem),
             new WaitCommand(initialWaitTime), // units in seconds
-            taxiRamseteCommand
-                .beforeStarting(
-                    () -> driveBaseSubsystem.resetOdometry(taxiTrajectory.getInitialPose()))
-                .andThen(() -> driveBaseSubsystem.stopDriveMotors()));
+            taxiRamseteCommand.andThen(() -> driveBaseSubsystem.stopDriveMotors()));
 
-    if (allSubsystemsEnabled) {
+    if (shooterEnabled) {
       oneBallCommand =
           new SequentialCommandGroup(
               new DeployIntake(intakeSubsystem, cdsSubsystem),
               new WaitCommand(initialWaitTime),
               new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, false),
               new WaitCommand(Constants.delaytaxi),
-              oneBallRamseteCommand
-                  .beforeStarting(
-                      () -> driveBaseSubsystem.resetOdometry(oneBallTrajectory.getInitialPose()))
-                  .andThen(() -> driveBaseSubsystem.stopDriveMotors()));
+              oneBallRamseteCommand.andThen(() -> driveBaseSubsystem.stopDriveMotors()));
 
       twoBallParallel =
           new ParallelDeadlineGroup(
-              twoBallRamseteCommand[0].beforeStarting(
-                  () ->
-                      driveBaseSubsystem.resetOdometry(
-                          twoBallTrajectory[0].getInitialPose())), // go out to get ball
+              twoBallRamseteCommands[0], // travel to get ball
               new IntakeForwardCommand(intakeSubsystem, cdsSubsystem)
                   .andThen(() -> driveBaseSubsystem.stopDriveMotors()));
```

Comment:
It seems like this could be added to the ramsete command and have the same effect. Same with below in the twoBallCommand. stopDriveMotors() can probably be added to every ramsete command in your getRamsetCommand method. I don't see any cases where you wouldn't want to stop after a trajectory.